http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/CookiesIT.java

```diff
@@ -52,6 +53,26 @@ void testSameSiteAttributeAddedByVertxHttpExt() {
                 .cookie("vertx", detailedCookie().sameSite(sameSite).secured(true));
     }
 
+    @Test
+    void testCookiesSerialization() {
+        given()
+                .get("/cookie/newcookie-serialization")
+                .then()
+                .statusCode(200)
+                .body(stringContainsInOrder(String.format("\"name\":\"%s\"", TEST_COOKIE),
+                        String.format("\"value\":\"%s\"", TEST_COOKIE_VALUE)));
+    }
+
+    @Test
+    void testCookiesSerialization3() {
+        given().cookie(String.format("%s=\"%s\";", TEST_COOKIE, TEST_COOKIE_VALUE))
```

Comment:
this feels less readable then `.cookie("key", "value")` but please feel free to keep if you like it

Reply:
I change it back. Probably in long term the better readability is better